### PR TITLE
InstrumentList.ts

### PR DESCRIPTION
**Pull request by donbasuno**
Added InstrumentList as discussed

### CODE REVIEW

**Comment by espenove** [comment]
Can you remove $ from $type, inline with the other discussion?

File: src/examples/InstrumentList.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+interface InstrumentList extends Context {
+  type: 'fdc3.instrumentList',
+  id: {
+    instrumentListId?: string;
```
Comment (rikoe) [inline src/examples/InstrumentList.ts]: I have a question about the id here. Should we leave this out? I think the instrumentList should just be an array of instruments, it doesn't preclude people from adding an id. "InsturmentListId" is quite specific.

Comment (espenove) [reply src/examples/InstrumentList.ts]: Agree with Riko, to specific and we don't preclude from adding ID's.
Could in some scenarios make sense with a name as well

**Comment by espenove** [comment]
Resolved in https://github.com/FDC3/ContextData/pull/15

**Pull request closed without merging**